README.md

```diff
@@ -1,36 +1,9 @@
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
-
 <div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
-  <h3><b>Microverse README Template</b></h3>
+  <h3><b>WeLead Political Summit</b></h3>
 
 </div>
 
```

Comment:
- [ ] [**OPTIONAL**] Good job adding the video, however instead of putting it in your PR summary Kindly add a new section in your README file and add it there. Because the video is not for me :smile: , it for people who will visit your GitHub account :gift:. Make sure the video doesn't exceed 5 minutes :cookie:

Reply:
thank you for reviewing my code